.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,3 +1,10 @@
+Checklist:
+
+* [ ] I have updated the necessary documentation
+* [ ] I have updated the changelog, if necessary (CHANGELOG.md)
+* [ ] I have signed off all my commits as required by [DCO](../CONTRIBUTING.md#legal)
```

Comment:
Also, following the link above as rendered in this very PR, I find that the resulting URL:
https://github.com/procore-oss/blueprinter/CONTRIBUTING.md
returns a 404
The actual content is at:
https://github.com/procore-oss/blueprinter/blob/main/CONTRIBUTING.md

(thanks for finding that bug--I was planning to use relative URLs in another repo, and now I know what _won't_ work, lol)

Reply:
Fixed